packages/ui/src/components/va-radio/VaRadio.vue

```diff
@@ -1,116 +1,183 @@
 <template>
-  <label
-    class="va-radio"
+  <VaMessageListWrapper
     :class="computedClass"
+    :disabled="disabled"
+    :success="success"
+    :messages="messages"
+    :error="computedError"
+    :error-messages="computedErrorMessages"
+    :error-count="errorCount"
+    @blur="onBlur"
+    ref="container"
   >
-    <input
-      class="va-radio__input"
-      type="radio"
-      :checked="isActive"
-      :disabled="$props.disabled"
-      :readonly="$props.readonly"
-      :name="computedName"
-      :value="computedLabel"
-      :aria-checked="isActive"
-      :tabindex="tabIndexComputed"
-      @change="onClick"
-      @focus="onFocus"
+    <label
+      v-for="(option, index) in options"
+      :key="index"
+      class="va-radio va-radio__square"
+      :style="labelStyle"
     >
-
-    <span
-      aria-hidden="true"
-      class="va-radio__icon"
-      :style="iconComputedStyles"
-    >
-      <span
-        class="va-radio__icon__background"
-        :style="iconBackgroundComputedStyles"
+      <input
+        ref="input"
+        :style="inputStyle"
```

Comment:
I know it was as it before, but it would be nice to move styles to `v-bind` in css. I will make a different issue for it.